src/controllers/log.controller.ts

```diff
@@ -0,0 +1,141 @@
+import { logDB } from '../models/db.index';
+import { RequestHandler } from 'express';
+import createHttpError from 'http-errors';
+
+export const getLogs: RequestHandler = async (req, res, errFunc) => {
+    try {
+        const logs = await logDB.findAll();
```

Comment:
Do we want to send these in any sort of special way? say they are about a user object, do we want to query the user db and send that user data along side the error?

Reply:
For a get all that could ginormous like logs, we wanna be able to send a param that limits the query. Say ?limit with a default value of 100.

(req.query) Checks query string params, ex: ?id=12 Checks urlencoded body params